test-container/src/main/java/io/strimzi/StrimziKafkaContainer.java

```diff
@@ -22,6 +23,8 @@
 import java.util.Collections;
 import java.util.List;
 
+// "Redundant NullCheck because the InputStreamReader is inside the try catch block"
+@SuppressFBWarnings("RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE")
```

Comment:
While is the null check redundant? If it is easy to fix you should fix it instead of skipping the check for the whole class.

Reply:
Done :)